frontend/src/components/shared/StreamlitMarkdown/StreamlitMarkdown.tsx

```diff
@@ -50,6 +53,14 @@ import {
 } from "./styled-components"
 
 import "katex/dist/katex.min.css"
+import {
+  getMdBlue,
+  getMdCyan,
+  getMdOrange,
+  getMdRed,
+  getMdTeal,
+  getMdViolet,
+} from "../../../theme/index"
```

Comment:
nit: Use absolute import here instead of relative import

Reply:
It's fixed now 👍